server/controllers/operations.js

```diff
@@ -30,9 +30,9 @@ export function preloadOtherOperation(req, res, next, otherOperationID) {
 export async function update(req, res) {
     let attr = req.body;
 
-    // For now, we can only update the category id or operation type of an operation.
-    if (typeof attr.categoryId === 'undefined' && typeof attr.operationTypeID === 'undefined')
-        return sendErr(res, 'missing parameter categoryId or operationTypeID', 400, 'Missing parameter categoryId or operationTypeID');
+    // For now, we can only update the category id, operation type or custom label of an operation.
```

Comment:
We can remove the "For now," as we have so many different fields to update nowadays ;)